src/components/pages/Analytics/Analytics.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import TechStackGraph from './TechStackGraph';
+import './Analytics.css';
+
+const Analytics = () => {
+  return (
+    <div>
```

Comment:
I see we chose to use a `div` here to wrap these sibling elements. Can we use a fragment instead `<> </>`. 

My personal rule of thumb with divs is to use them semantically (when the element calls for a divider) or whenever I need to wrap some elements in a divider and add attributes via the className/id etc. for CSS selector/eventHandler purposes. I see there are not props/attributes on this div wrapping the sibling divs. So I'd suggest a fragment here as it is a part of the React API.

Reply:
Totally makes sense, thanks Ryan!